openassessment/assessment/models/peer.py

```diff
@@ -108,7 +109,7 @@ class PeerWorkflow(models.Model):
 
     """
     # Amount of time before a lease on a submission expires
-    TIME_LIMIT = timedelta(hours=8)
+    TIME_LIMIT = timedelta(hours=getattr(settings, "PEER_LEASE_EXPIRATION_HOURS", 8))
```

Comment:
@0x29a Could you prefix these two new settings with `ORA_` so we know what app they affect?

I also think we need to document these as `SettingDictToggle`s in edx-platform somewhere so operators know they exist. I think they can go in [cms.djangoapps.contentstore.toggles](https://github.com/openedx/edx-platform/blob/master/cms/djangoapps/contentstore/toggles.py).

Reply:
@pomegranited, good catch, thanks. Fixed.

>I also think we need to document these as SettingDictToggles in edx-platform somewhere so operators know they exist. I think they can go in [cms.djangoapps.contentstore.toggles](https://github.com/openedx/edx-platform/blob/master/cms/djangoapps/contentstore/toggles.py).

Hmm, this setting isn't a toggle and it's not CMS-specific. There are other ORA-prefixed settings here: https://docs.openedx.org/projects/edx-platform/en/latest/references/settings.html, which is auto-generated from `common.py`. Perhaps we can add it there?

Reply:
Ah good points that this isn't a "toggle", and it's not used in the CMS. If you can add it to the lms.env.common and get into those docs, that would be great -- doesn't have to block merging this though, so I'll do that today.